bentoml/_internal/runner/runnable.py

```diff
@@ -135,7 +135,9 @@ def get_batch_dim(idx: int) -> int:
 
         self.args = []
         self.kwargs = {}
-        for i, (name, param) in enumerate(params.items()):
+        # drop self parameter from params
+        param_items = list(params.items())[1:]
```

Comment:
not all the cases have self as the first arg

Reply:
I think it's ok to not support `staticmethod` here, though we may want to have better error messages for those cases.